cert-update.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+certbot renew
+cd /api-certificates
+git pull
+tar cvf /api-certificates/letsencrypt_backup.tar /etc/letsencrypt
+git add .
+git commit -m "automated upload from google cloud"
```

Comment:
it may be worth getting the current date in that commit name, but I would not include it in this PR, it already seems big enough as is

Reply:
agreed